docs/resources/aws_cloudfront_distribution.md

```diff
@@ -102,6 +110,24 @@ Use `should_not` to test the entity should not exist.
       it { should_not exist }
     end
 
+### s3\_origin\_path
+
+The `s3_origin_path` either return origin path or blank in case of no origin configured.
+
+      describe aws_cloudfront_distribution(distribution_id: 'EXISTING_DISTRIBUTION_ID', origin_domain_name: 'EXISTING_CF_ORIGIN_NAME_1') do
+        its ('s3_origin_path') { should include '/next' }
+      end
+
+      describe aws_cloudfront_distribution(distribution_id: 'EXISTING_DISTRIBUTION_ID', origin_domain_name: 'EXISTING_CF_ORIGIN_NAME_2') do
+        its ('s3_origin_path') { should include '/release' }
+      end
+
+For default origin path
+
+    describe aws_cloudfront_distribution(distribution_id: 'EXISTING_DISTRIBUTION_ID', origin_domain_name: 'EXISTING_CF_ORIGIN_NAME_3') do
+        its ('s3_origin_path') { should include '' }
+    end
```

Comment:
```suggestion
Use `s3_origin_path` to return an origin path for the specified origin domain name if the origin path is configured, otherwise it returns an empty string.

      describe aws_cloudfront_distribution(distribution_id: 'DISTRIBUTION_ID', origin_domain_name: 'ORIGIN_DOMAIN_NAME') do
        its ('s3_origin_path') { should include '/next' }
      end

      describe aws_cloudfront_distribution(distribution_id: 'DISTRIBUTION_ID', origin_domain_name: 'ORIGIN_DOMAIN_NAME') do
        its ('s3_origin_path') { should include '/release' }
      end

For the default origin path:

    describe aws_cloudfront_distribution(distribution_id: 'DISTRIBUTION_ID', origin_domain_name: 'ORIGIN_DOMAIN_NAME') do
        its ('s3_origin_path') { should include '' }
    end
```